components/modal/Register/register.js

```diff
@@ -118,7 +127,7 @@ const Register = (props) => {
           id="reason"
           className={classNames.inputBox}
           type="text"
-          placeholder="Your Reason why you are interested on this candidate(Max 100 characters)"
+          placeholder="Why you are interested on this candidate(Max 100 characters)"
```

Comment:
Is this grammatically correct ? 
Does it looks nice to add the character count in the UI ? @DashDeipayan